htdocs/takepos/reduction.php

```diff
@@ -197,14 +202,33 @@ function ValidateReduction()
 			});
 		} else {
 			console.error('Error bad reduction type :', reductionType);
+			return false
+		}
+
+		return true;
+	}
+
+	$(document).ready(function() {
+		// focus on manual input
+		$("#reduction_total").focus();
+	});
+
+	// manual input validation
+	function formvalid(type) {
+		reductionType = type;
+		if (reductionType != "") {
+			return ValidateReduction();
 		}
+		return false;
 	}
 </script>
 
 <div style="position:absolute; top:2%; left:5%; width:91%;">
 <center>
 <?php
-	print '<input type="text" class="takepospay" id="reduction_total" name="reduction_total" style="width: 50%;" placeholder="'.$langs->trans('Reduction').'">';
+	print '<input type="text" class="takepospay" id="reduction_total" name="reduction_total" size="10" placeholder="'.$langs->trans('Reduction').'">';
```

Comment:
Attribute "size" is deprecated in html5. Must use css or embedded style.

Reply:
@eldy done; thanks for the autofocus that i did no know... time for me to update my skills or/and to hire younger developers than me ;)